py_zipkin/util.py

```diff
@@ -58,3 +58,11 @@ def signed_int_to_unsigned_hex(signed_int):
     if hex_string.endswith('L'):
         return hex_string[:-1]
     return hex_string
+
+
+def _should_sample(sample_rate):
```

Comment:
similar to above, this shouldn't be a part of a change around parsing..

Reply:
similar to above, I think it's pretty good and don't understand the rationale for not having the `ZipkinAttrs` that come out being ready to feed into `zipkin_span()`.